### DISCUSSION TITLE
Can it monitor .onion and .i2p sites?

### DISCUSSION THREAD

**Discussion by 002gRTEah3mRYH** [upptime/upptime#685] [Q&A]
Hello,

Can it monitor .onion and .i2p sites?
If not, what are the tools for?

Thanks

**Comment by AnandChowdhary** [comment]
I'm not familiar with this. But I think we can somehow use the `CURLPROXY_SOCKS5_HOSTNAME` setting of cURL to make requests to .onion domains?
